Cirrious.FluentLayout/FluentLayout.cs

```diff
@@ -153,7 +154,7 @@ private void ThrowIfSecondItemAlreadySet()
 
         public IEnumerable<NSLayoutConstraint> ToLayoutConstraints()
         {
-            var constraint = NSLayoutConstraint.Create(
+            Constraint = NSLayoutConstraint.Create(
```

Comment:
> I don´t see an easier (with less code) way to gain access to NSLayoutConstraints

This method is already returning the constraint, so you could access it that way? Is that not an option for you for some reason? I've done exactly that in apps before.

I certainly agree that it's not the most elegant solution, but given that there is still a pretty straightforward workaround for consumers of the API I'd rather try and figure out a better class design as a means of solving this at some point, rather than make the API more confusing.